asu/util.py

```diff
@@ -368,6 +368,21 @@ def parse_feeds_conf(url: str) -> list[str]:
     )
 
 
+def is_new_style_build(version: str) -> bool:
```

Comment:
I think this naming is a bit.. unspecific. Isn't something like `is_snapshot` better for this?